core/style.js

```diff
@@ -1462,10 +1462,10 @@ CKEDITOR.STYLE_OBJECT = 3;
 
 			if ( docFrag ) {
 				var newBlockClone = newBlock.clone();
-				newBlockClone.setHtml( blockHtml );
+				newBlockClone.setHtml( CKEDITOR.tools.htmlSafeByReview(blockHtml, 'blockHtml is created internally and is only impacted by the editors config options') );
```

Comment:
Can we bless preHtmls before they get passed into this function?

Reply:
Will be blessed at last modification instead